sdk/keyvault/azure-security-keyvault-keys/src/samples/java/com/azure/security/keyvault/keys/KeyAsyncClientJavaDocCodeSnippets.java

```diff
@@ -497,6 +498,28 @@ public void listKeyVersions() {
         // END: com.azure.security.keyvault.keys.async.keyclient.listKeyVersions
     }
 
+    /**
+     * Generates code samples for using {@link KeyAsyncClient#getRandomBytes(int)} and
+     * {@link KeyAsyncClient#getRandomBytesWithResponse(int)}.
+     */
+    public void getRandomBytes() {
+        KeyAsyncClient keyAsyncClient = createAsyncClient();
+
+        // BEGIN: com.azure.security.keyvault.keys.KeyAsyncClient.getRandomBytes#int
+        int amount = 16;
```

Comment:
Should this be `count` to be consistent with the method signature?

Reply:
Yes, definitely.